libs/template/templates/default-python/databricks_template_schema.json

```diff
@@ -29,6 +29,13 @@
             "enum": ["yes", "no"],
             "description": "Include a stub (sample) Python package in '{{.project_name}}{{path_separator}}src'",
             "order": 4
+        },
+        "serverless": {
+            "type": "string",
+            "default": "no",
```

Comment:
Why do we use `no` as default by the way?

Reply:
I think this should be yes since that's what we are promoting as a company. cc: @lennartkats-db @pietern

Reply:
I don't disagree, but it can be a separate PR or we can decide to do it in a later version.

I'm also working on integration tests for standard templates (with deploy & summary & run), those would give more confidence in the new one.